bt_level_order_traversal.java

```diff
@@ -0,0 +1,74 @@
+/**
+BFS Solution
+        
+        Time Complexity: O(N)
+        Space Complexity: O(1) constant.
```

Comment:
Please recheck space complexity (recursive stack space or queue). It is not constant.